pkg/yurtmanager/controller/platformadmin/platformadmin_controller.go

```diff
@@ -393,6 +393,9 @@ func (r *ReconcilePlatformAdmin) reconcileComponent(ctx context.Context, platfor
 	defer func() {
 		platformAdminStatus.ReadyComponentNum = readyComponent
 		platformAdminStatus.UnreadyComponentNum = int32(len(platformAdminFramework.Components)) - readyComponent
+		if platformAdminStatus.UnreadyComponentNum != 0 {
+			platformAdminStatus.Ready = false
```

Comment:
It would be more appropriate to use "return false, nil" here. Because if it returns true, when it gets back to the reconcileNormal function, it will continue to execute further, setting platformAdminStatus.Ready to true.